conf/configuration.go

```
@@ -382,6 +384,22 @@ func validatePlaylistsPath() error {
 	return nil
 }
 
+func validatePurgeMissingOption() error {
+	allowedValues := []string{"never", "always", "full"}
+	valid := false
+	for _, v := range allowedValues {
+		if v == Server.Scanner.PurgeMissing {
+			valid = true
+			break
+		}
+	}
+	if !valid {
+		log.Error(fmt.Sprintf("Invalid Scanner.PurgeMissing value: '%s'. Must be one of: %v", Server.Scanner.PurgeMissing, allowedValues))
+		Server.Scanner.PurgeMissing = "never"
```

We should return an error if the value is invalid